backend/src/main/java/fk/prof/backend/model/policy/PolicyStore.java

```diff
@@ -0,0 +1,22 @@
+package fk.prof.backend.model.policy;
+
+import fk.prof.backend.proto.BackendDTO;
+import recording.Recorder;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * TODO: Liable for refactoring. Dummy impl for now
+ */
+public class PolicyStore {
```

Comment:
This abstraction is useless as of now, mentioned in comment above -> dummy impl. I expect this to be replaced with a policy store impl backed by zookeeper.